syncd/syncd_flex_counter.cpp

```diff
@@ -208,53 +219,58 @@ void FlexCounter::removePort(
     // Remove flex counter if counter IDs map is empty
     if (fc.m_queueCounterIdsMap.empty() && fc.m_portCounterIdsMap.empty() && fc.m_queueAttrIdsMap.empty())
     {
-        removeInstance(pollInterval);
+        removeInstance(instanceId);
     }
 }
 
 void FlexCounter::removeQueue(
```

Comment:
for removeQueue, I think you changed the logic.

In marian's code, he tries to remove the queue id both from m_queueCounterIdsMap and m_queueAttrIdsMap. 

In your code, if the first one is not found, you will return and will not remove the queueid from the second map.

Why do you change the logic here? Is this intended? IMO, marian code is more correct.

Reply:
Marian's new logic was added recently and I didn't rebase it before force push. Will update and sorry for the inconvenience